pkgs/by-name/pi/pinact/package.nix

```diff
@@ -15,7 +15,7 @@ let
     hash = "sha256-OQo21RHk0c+eARKrA2qB4NAWWanb94DOZm4b9lqDz8o=";
   };
 in
-buildGo120Module {
+buildGoModule {
```

Comment:
New recommended way since https://github.com/NixOS/nixpkgs/pull/280843? 👀

Reply:
- They are actually using Go 1.21 in the build step, it will fit for current `buildGoModule` than now. https://github.com/suzuki-shunsuke/pinact/blob/84d8bc0489a5bc286b4442b7990adc8d2e9d4a54/.github/workflows/release.yaml#L12
- I have tested the actual behavior with `nix run github:aaronjheng/nixpkgs/86025874d5c303d1c024ef4a61477e360cff84f8#pinact -- run`